base/atomics.jl

```diff
@@ -462,7 +329,4 @@ fences should not be necessary in most cases.
 
 For further details, see LLVM's `fence` instruction.
 """
-atomic_fence() = llvmcall("""
-                          fence seq_cst
-                          ret void
-                          """, Cvoid, Tuple{})
+atomic_fence() = Core.Intrinsics.atomic_fence(:sequentially_consistent)
```

Comment:
should we change this to the following?
suggestion```
atomic_fence(mode::Symbol=:sequentially_consistent) = Core.Intrinsics.atomic_fence(mode)
```

Reply:
broken markdown